### ISSUE TITLE
Auto-complete hides caret and typing after hitting enter

### ISSUE THREAD

**Issue by redtopia** [jcberquist/sublimetext-cfml#92]
Using the following example, auto-complete often hides the next line I'm working on after I hit ENTER because it doesn't preposition itself correctly. It will sit on top of the line I'm typing, preventing me from seeing what I type. It will usually stays that way unless I do something like hit ESC or click away.

```
var results = queryExecute(
    sql="
        SELECT *
        FROM Users
        WHERE userID = :userID
    ",
    params=[
        {name="userID", value=userID, cfsqltype="cf_sql_bigint"}
    ],
    options={
        datasource: dsn
    }
);

```
To reproduce the behavior, open up an editor and manually type in the example above.

Although I like the auto complete feature, when it hides my typing, it's incredibly annoying and difficult to work with. What I think needs to happen is when the user hits enter, auto complete should go away, and then pop up again if the user starts typing. If this can't be fixed, please advise how to turn off auto-complete, and if possible, could I get it to open using a command key?

**Comment by jcberquist** [comment]
If you are referring to the doc popup, and not the auto complete suggestions themselves, you can turn it off by setting `"cfml_completion_docs": false` in your CFML settings.

**Comment by redtopia** [comment]
No, I turned that off already and assigned a keyboard shortcut for it. I’m talking about the code completion popup.

**Comment by jcberquist** [comment]
Hmm, I don't see the behavior you are describing...I am probably still not understanding you. In any case, you can turn off auto complete entirely by setting `"auto_complete": false` in your global user settings. You can still trigger it with `CTRL + SPACE` in such a case. Is that what you are looking for?

**Comment by redtopia** [comment]
Here's an animation of the behavior:

http://g.recordit.co/GfdoBqiXpe.gif

**Comment by redtopia** [comment]
Is that something that your plugin can fix, or is that a ST3 issue?

**Comment by jcberquist** [comment]
Ok, that is what I was referring to earlier - you can turn that popup off with `"cfml_completion_docs": false`

**Comment by redtopia** [comment]
OK, that's much better! If there any way to get it fixed, I would probably turn it back on. Thanks!
